client/lobby/SelectionTab.cpp

```diff
@@ -534,6 +534,13 @@ void SelectionTab::parseMaps(const std::unordered_set<ResourceID> & files)
 	{
 		try
 		{
+			const std::string filename_prefix = file.getName().substr(file.getName().find_last_of("/\\") + 1, 2);
+			if(filename_prefix == "__")
```

Comment:
I presume this is to hide campaign maps?
We can also use different directory for them - only maps from "Maps" directory are visible in selection tab
a) use existing Data directory
b) create separate directory for campaigns, e.g. Campaigns and load campaigns & their maps from there

Or another option - use different extension for campaign scenario so selection tab won't pick them up

Reply:
or maybe use maps in zip package for campaigns like map editor does for maps?

Reply:
I was thinking that custom campaigns shall be as separate mods.
Campaign mod can contain custom data, music, objects, specific configurations and of course maps.
The only thing is that we don't want to show those maps in scenario list.

Reply:
We already have launcher to as content distributing system, so better to use that, than create something new

Reply:
I rather see campaigns requiring mods as something similar to maps requiring mods, with exception that campaigns will appear on custom campaign list rather than on map list

Reply:
also what about reusing campaign content in map editor and random map generator etc.

Reply:
Fixed